theme/type/baselineType.less

```diff
@@ -2,11 +2,16 @@
 
 @import (reference) "../layout/grid";
 @import (reference) "type";
+@import (reference) "../mixins/type";
 
-.__baselineType(@type-scale; @row-span) when (isnumber(@type-scale)) and (isnumber(@row-span)) {
+.__baselineType(@type-scale; @row-span; @baseline) when (isnumber(@type-scale)) and (isnumber(@row-span)) and (@baseline = true) {
```

Comment:
Yep this and your other comment is what I was getting at not 100% happy... I'll chat to you on Slack.